stock_picking_invoice_link/models/stock_picking.py

```diff
@@ -17,9 +17,35 @@ class StockPicking(models.Model):
     invoice_id = fields.Many2one(
         comodel_name='account.invoice', string='Invoice',
         compute="_compute_invoice_id")
+    invoice_exists = fields.Boolean(
```

Comment:
Although not exactly accurate, I will use picking invoice_status for showing the button.